modules/caddyhttp/map/map.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2015 Matthew Holt and The Caddy Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mmap
+
+import (
+	"net/http"
+	"regexp"
+
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
+)
+
+func init() {
+	caddy.RegisterModule(Handler{})
+}
+
+// Handler - Map
+//
+//
+type Handler struct {
+	Source      string `json:"source,omitempty"`
+	Destination string `json:"destination,omitempty"`
+	Default     string `json:"default,omitempty"`
+	Items       []Item `json:"items,omitempty"`
+}
```

Comment:
Fill out godoc. How does this work? What does it do? Why would someone use it?

Struct tags look good.

Reply:
Just a note that it still needs godoc :)

Reply:
Sorted I hope :)